docker2earthly/convert.go

```diff
@@ -180,7 +180,7 @@ type earthfileTemplateArgs struct {
 }
 
 // GenerateEarthfile returns an Earthfile content string which contains a target to build a docker image using FROM DOCKERFILE
-func GenerateEarthfile(buildContextPath string, dockerfilePath string, imageTags []string, buildArgs []string, platforms []string, target string) (string, error) {
+func GenerateEarthfile(buildContextPath string, dockerfilePath string, imageTags []string, buildArgs []string, platformsStr string, target string) (string, error) {
```

Comment:
this feels inconsistent -- why should imageTags and buildArgs be passed as a slice, but platforms be passed as a comma separated string?

I think it would be better to expose `splitPlatforms` and have the caller of `GenerateEarthfile` pre-split the platforms.

Reply:
furthermore, it looks like it's also possible to pass `--platform` as separate flags, e.g. `docker build --platform linux/amd64 --platform linux/arm64` so we should be able to support this as well.

Reply:
@alexcb , updated, please let me know if that looks good to you.

Reply:
looks great! Plus you wrote a `TestSplitFlagString` :raised_hands: